### PR TITLE
nvchecker 2.8.post1

### PR DESCRIPTION
**Pull request by peeweep**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Is it just needed to test?

File: pypi_formula_mappings.json [hunk 1]
```diff
@@ -230,6 +230,9 @@
   "nox": {
     "exclude_packages": ["six"]
   },
+  "nvchecker": {
+    "extra_packages": ["packaging"]
+  },
```
Comment (cho-m) [inline pypi_formula_mappings.json:235]: Probably better to enable feature directly.
```suggestion
  "nvchecker": "nvchecker[pypi]",
```

**Comment by peeweep** [comment]
> Is it just needed to test?

My description in commit [538dc35](https://github.com/Homebrew/homebrew-core/pull/99103/commits/538dc35) is inaccurate, `packaging` is required when there is `source = "pypi"` in the toml config file.

File: pypi_formula_mappings.json (hunk 1 above)
Comment (peeweep) [reply pypi_formula_mappings.json]: Thanks, it looks better. Why put `nvchecker[pypi]` here? Is there documentation?

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**